### ISSUE TITLE
MissingParameterName false positive on multiline array type syntax

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#753]
When using multiline array documenting syntax
```php
/**
 * @param array{
 *     asn: int
 *     } $x
 */
public function a(array $x) : void
{
}
```

Squiz.Commenting.FunctionComment.MissingParamName sniff does not detect parameter name

> Missing parameter name

**Comment by kukulich** [comment]
@simPod What version do you use?

**Comment by simPod** [comment]
Ooooh, different repo.

lastest master. But should close and move to squiz
